pkg/operator/reports.go

```diff
@@ -734,6 +740,73 @@ func (op *Reporting) runReport(logger log.FieldLogger, report *metering.Report)
 	return nil
 }
 
+func deleteReport(logger log.FieldLogger, report *metering.Report, op *Reporting) {
```

Comment:
Is this supposed to be a method of the `Reporting` structure? I see `op.deleteReport` referenced throughout, but I don't see that defined anywhere.

Reply:
this PR only references this new `deleteReport()`, a function on the default class for the file `reports.go`, not sure how to refer to it. It's at the same location as for instance: `getReportPeriod()` function. Should it be moved to be a method of the Reporting type?

Reply:
moved and using the op method now